### PR TITLE
New post: Schnorr Basics

### PR DESCRIPTION
**Pull request by kallerosenbaum**

### CODE REVIEW

File: content/blog/schnorr-basics/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  - name: Kalle Rosenbaum
+    github: kallerosenbaum
+    twitter: kallerosenbaum
+tags:
+  - "Schnorr"
+  - "signature"
+  - "BIP340"
+mathjax: true
+images:
+    - "schnorr-basics/sig-overview.png"
```
[the remark this answers is not in the document]
Comment (0xB10C) [reply content/blog/schnorr-basics/index.md:15]: also re-exported the title image in a 2:1 width/height ratio. This makes it align more with the other title images.

before:
image
after:
image

[the remark this answers is not in the document]
Comment (0xB10C) [reply content/blog/schnorr-basics/index.md:15]: @kallerosenbaum this has your twitter avatar image in it now (similar to elle's and my post). Let me know if I should drop it.

**Review by 0xB10C — COMMENTED** [review]
Thank you for cross-posting this. I appreciate your effort!

A few comments below, but I figured it's easier/less work for you if I just fix them up myself. Feel free to squash all `fixup:...` commits in [dev-bitcoin/blog/2022-01-schnorr-basics-fixups](https://github.com/dev-bitcoin/blog/tree/2022-01-schnorr-basics-fixups) into your commit. You can drop my commit messags when doing so.

I've also ran this through grammarly and it suggested a few additional commas in some places. If you want, I can add a commit fixing up some of these too.

**Comment by kallerosenbaum** [comment]
All are good changes, thanks! Please incorporate the Grammarly changes too, if you find that they improve things.

**Comment by 0xB10C** [comment]
The low-hanging grammarly suggestions here without cutting to much into your writing style:

https://github.com/dev-bitcoin/blog/tree/2022-01-schnorr-basics-grammar-fixups

**Comment by kallerosenbaum** [comment]
Thanks! Pushed all but one change. I kept the second comma on this line:

> didn't have $P$ in the challenge, $e=H(R||m)$, and that the signature

**Comment by 0xB10C** [comment]
Awesome, thanks. Will read through it one more time today or tomorrow and then merge & tweet about it tomorrow.

Here's a deployment preview: https://deploy-preview-9--bitcoin-dev-blog-preview.netlify.app/

**Review by Naruto0-sudoy — COMMENTED** [review]
thalindaAC2218918

**Pull request merged**